exercises/robot-simulator/example.cpp

```diff
@@ -0,0 +1,87 @@
+#include "robot_simulator.h"
+
+robot_simulator::Robot::Robot() : robot_position(std::make_pair(0, 0)), robot_bearing(robot_simulator::Bearing::NORTH) {}
```

Comment:
I would define this file inside the namespace to avoid these long names.

Reply:
Good point, I've done as you're suggesting.